recipes/norm/all/conanfile.py

```diff
@@ -1,7 +1,8 @@
 from conans import ConanFile, CMake, tools
 import os
-import glob
-import yaml
+
+
+required_conan_version = ">=1.22.0"
```

Comment:
This feature was no added to 1.26 so it can be removed

```suggestion
```

Reply:
I see in the code and the example! I was looking at the signature of get() in the documentation and there it pops up after 1.22. Cool, thanks!